index.d.ts

```diff
@@ -34,18 +34,44 @@ declare namespace pug {
   export function compileFileClient(path: string, options?: Options): string;
 
   /**
+   * Compile a Pug template and render it without locals to html string.
+   */
+  export function render(template: string): string;
+  /**
    * Compile a Pug template and render it with locals to html string.
+   * @param {(Options & LocalsObject)} options Pug Options and rendering locals
```

Comment:
It's not about looking nice, and it is already useful. It's more about maintaining something. Although this isn't a big use-case, there's dozens of other people with huge definitions who are trying to maintain them. If it takes a cumulative 5 minutes extra each time to update all definitions because we repeated ourselves, that's 100s of hours over a long enough duration and, of course, the probability that each change could introduce a bug because a new contributor doesn't realise this is required or we forget. 

I found and commented on https://github.com/Microsoft/TypeScript/issues/407.